app.py

```diff
@@ -52,34 +52,51 @@ def menu_loop():
         choice = input('Action: ').lower().strip()
 
         if choice in menu:
-            clear()
             menu[choice]()
 
-menu = OrderedDict([
-    ('a', add), 
-    ('v', view),
-    ('b', backup),
-    ('d', delete), 
-])
 
-
-def view():
-    '''View items in database''' # TODO
+def view(search_query=None):
+    """View items in inventory."""
+    products = Product.select().order_by(Product.timestamp.desc())
+    if search_query:
+        product = products.where(Product.product_name.contains(search_query))
     pass
 
 
+def add():
+    """Add items to inventory."""
+    print("Enter a product below, press ctrl + d when finished.\n")
+    print("Format for entry: Name, Quantity, Price (0.00)")
+    data = sys.stdin.read().strip()
+
+    if data:
+        if input('Save Entry? [Yn]').lower() != 'n':
+            Product.create(
+                product_name=data,
+                product_quantity=data,
+                product_price=data,
+            )
```

Comment:
```suggestion
    if data:
        if input('Save Entry? [Yn]').lower() != 'n':
            Product.create(
                product_name=data,
                product_quantity=data,
                product_price=data,
            )
``` Need to find out how to take data and convert to a int.

Reply:
From office hours: " I would encourage you to do a `print(data) ` and see why this is failing :slightly_smiling_face:  I think it will become obvious.
Also, I would point out that what you probably want here is mulitple inputs. One for each thing :slightly_smiling_face: ie one for the name, one for the quantity, and one for the price"